systemtest/apmservertest/server.go

```diff
@@ -288,7 +288,7 @@ func (s *Server) consumeStderr(procStderr io.Reader) {
 	s.Stderr = stderrPipeReader
 
 	type logEntry struct {
-		Timestamp logpTimestamp `json:"@timestamp"`
+		Timestamp logpTimestamp `json:"timestamp"`
```

Comment:
```suggestion
		Timestamp logpTimestamp `json:"@timestamp"`
```

Sorry I missed this earlier, I don't think this one should have changed. It's `@timestamp` in ECS, and I checked out the PR locally and it's indeed `@timestamp` in the logs.

Reply:
good catch